modules/planning/reference_line/BUILD

```diff
@@ -127,7 +127,7 @@ cc_test(
 
 cc_test(
     name = "discrete_points_reference_line_smoother_test",
-    size = "small",
+    size = "large",
```

Comment:
Ditto. Please justify why "small" or "medium" is not enough ?

Reply:
I have tested, at least two minutes